api/controllers/site.js

```diff
@@ -24,25 +25,26 @@ module.exports = {
     let site;
 
     Promise.resolve(Number(req.params.id)).then((id) => {
-      if (isNaN(id)) {
+      if (_.isNaN(id)) {
```

Comment:
is this necessary? this style isNaN is not used throughout the codebase

Reply:
The `isNaN` was throwing linting error's so I used the underscore version.